layouts/shortcodes/http-api.html

```diff
@@ -23,4 +23,7 @@
 {{ $base_url := (index $api_data.servers 0).variables.basePath.default }}
 {{ $path := delimit (slice "api" $spec $api) "/" }}
 
-{{ partial "openapi/render-api" (dict "api_data" $api_data "base_url" $base_url "path" $path) }}
+{{ $api_data = partial "json-schema/resolve-refs" (dict "schema" $api_data "path" $path) }}
+{{ $api_data := partial "json-schema/resolve-allof" $api_data }}
```

Comment:
why do we need to call `resolve-allof` here now?

Reply:
Sorry, it must be a leftover of when I was trying stuff.

Reply:
I removed it.